reports/config-example.ini

```
@@ -0,0 +1,24 @@
+[DEFAULT]
+year = 2021
+month = 12
+month_name = December
+email_from = general+cal-itp@jarv.us
```

Please update this to:

```suggestion
email_from = hello@calitp.org
```